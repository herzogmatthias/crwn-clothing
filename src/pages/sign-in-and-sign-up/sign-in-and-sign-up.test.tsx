import React from "react";
import { ShallowWrapper, shallow } from "enzyme";
import SignInAndSignUp from "./sign-in-and-sign-up.component";

describe("SIGN-IN-AND-SIGN-UP TESTS", () => {
  let wrapper: ShallowWrapper = shallow(<SignInAndSignUp></SignInAndSignUp>);

  test("render SignInAndSignUp Component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
