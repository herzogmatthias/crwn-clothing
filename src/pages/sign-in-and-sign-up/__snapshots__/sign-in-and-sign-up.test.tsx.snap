// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SIGN-IN-AND-SIGN-UP TESTS render SignInAndSignUp Component 1`] = `
<styled.div>
  <Connect(SignIn) />
  <Connect(SignUp) />
</styled.div>
`;
